src/stubs/aliases

```diff
@@ -44,3 +44,7 @@ function serve-proxy() {
         echo "  serve-proxy domain port"
     fi
 }
+
+function artisan() {
+    php artisan $1 $2
```

Comment:
I believe there is a way in Bash to just expand out all arguments instead of limiting it to the first 2.